python/manageJobs.py

```diff
@@ -260,7 +280,39 @@ def manageJobs(argv=None):
                 # actions that can be applied to all jobs
                 jobnums = [j.num for j in jobs]
                 schedd.act(htcondor.JobAction.Remove,jobnums)
+            elif options.xrootdResubmit:
+                print ""
+                file_and_site_per_file = find_input_file_site_per_job(argv = (["-c", options.inputFileClassAd] if options.inputFileClassAd else []) +
+                                                                             (["-k", options.logKey] if options.logKey and options.logPath else []) +
+                                                                             (["-l", options.logPath] if options.logKey and options.logPath else []) +
+                                                                             (["-u"] if options.preferUSSites else []) +
+                                                                             options.xrootdResubmitOptionsOther.split(), condor_jobs = jobs);
 
+                jobs_resubmitted = {}
+                jobs_not_resubmitted = {}
+                print "Resubmitting jobs (dryRun = " + str(options.dryRun) + ") ...",
+                sys.stdout.flush()

```

Comment:
in other contexts i define:
```python
def fprint(msg):
    import sys
    print(msg)
    sys.stdout.flush()
```
to make python's stdout behave sensibly

Reply:
Implemented